indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,128 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelection() {
+        $('.sessions-wrapper').on('change', 'table.i-table input.select-row', function() {
```

Comment:
wouldn't `.select-row` be enough of a selector? huge selectors are usually more expensive to evaluate, and I guess `.select-row` is clear enough